doc/package-specific-user-notes.xml

```diff
@@ -398,9 +398,8 @@ overrides = self: super: rec {
    <para>
     The FHS-compatible chroot used for steam can also be used to run other
     linux games that expect a FHS environment. To do it, add
-<programlisting>pkgs.(steam.override {
+<programlisting>(pkgs.steam.override {
           nativeOnly = true;
-          newStdcpp = true;
```

Comment:
as [pkgs/top-level/all-packages.nix](https://github.com/NixOS/nixpkgs/blob/a8cc2691f503ea13e146717607318b2be72ac252/pkgs/top-level/all-packages.nix) contains

```nix
  steam-run-native = (steam.override {
    nativeOnly = true;
  }).run;
```
this can probably be simplified even more ...

Reply:
Thanks! Updated the PR.